ReactiveCocoa/WatchKit/WKInterfaceButton.swift

```diff
@@ -3,7 +3,7 @@ import WatchKit
 
 extension Reactive where Base: WKInterfaceButton {
 	/// Sets the title of the button.
-	public var text: BindingTarget<String?> {
+	public var title: BindingTarget<String?> {
```

Comment:
I have deprecated the `text` attribute in favour of `title` (to match the WatchKit name), is that ok?

Reply:
Great 👍